### PR TITLE
Linkage on load

### PR DESCRIPTION
**Pull request by mennoo1996**
Related to #151, but couldn't reopen that one because of force push
Fixes link between director shot and camera shots on load.
#151 contained a comment about a used offset. This is still there, because it is needed to load blocks correctly, but these offsetted blocks are now auto deleted when load is finished.

### CODE REVIEW

**Comment by Markvanderuit** [comment]
I'll wait for a second member to confirm before merging, but the code looks good.

File: src/main/java/control/ProjectController.java [hunk 1]
```diff
@@ -365,6 +377,31 @@ public void load() {
...
+     */
+    private void removeRedundantCameraBlocks(ScriptingProject project) {
+        for (int i = 0;i < project.getDirectorTimeline().getShots().size();i++) {
+            DirectorShot dShot = project.getDirectorTimeline().getShots().get(i);
+            dShot.getCameraShots().forEach(cameraShot -> {
+                    for (int j = 0;j < project.getCameraTimelines().size();j++) {
+                        CameraTimeline timeline = project.getCameraTimelines().get(j);
+                        timeline.getShots().forEach(shot -> {
+                                if (cameraShot.getInstance() == shot.getInstance()) {
+                                    //timeline.removeShot(shot);
```
Comment (geenen124) [inline src/main/java/control/ProjectController.java]: Please remove commented code

File: src/main/java/control/ProjectController.java [hunk 2]
```diff
@@ -365,6 +377,31 @@ public void load() {
...
+                                    shot.setBeginCount(UNUSED_BLOCK_OFFSET);
+                                    shot.setEndCount(UNUSED_BLOCK_OFFSET);
+                                }
+                            });
+                    }
+                });
+        }
+        
+    }
+
```
Comment (geenen124) [inline src/main/java/control/ProjectController.java:401]: Unnecessary whitespace?

File: src/main/java/control/TimelineController.java [hunk 3]
```diff
@@ -159,6 +159,10 @@ public void removeCameraShot(CameraShot shot) {
      */
     public void shotChangedHandler(CameraShotBlockUpdatedEvent event) {
         CameraShotBlock changedBlock = event.getCameraShotBlock();
+        cameraShotBlocks.forEach(shotBlock -> {
+
```
Comment (geenen124) [inline src/main/java/control/TimelineController.java]: Unnecessary whitespace?

File: src/main/java/data/ScriptingProject.java [hunk 4]
```diff
@@ -109,6 +110,30 @@ public void saved() {
...
    
+   
+    
+    /**
+     * Get the maximum instance used so far.
+     * @return the maximum instance used so far
+     */
+    public int getMaxInstance() {
+        int res = Integer.MIN_VALUE;
+        for (int i = 0;i < directorTimeline.getShots().size();i++) {
```
Comment (geenen124) [inline src/main/java/data/ScriptingProject.java]: For loop can be changed to a forEach
